### DISCUSSION TITLE
Problem with exporting Dataview content

### DISCUSSION THREAD

**Discussion by GeroldC** [l1xnan/obsidian-better-export-pdf#145] [Q&A]
Hello there!
I have a problem with exporting a Dataview-Table.

I've created a movie database and I want to print it as a PDF.

![Screenshot 2024-03-15 002542](https://github.com/l1xnan/obsidian-better-export-pdf/assets/114219038/d4d35662-9f4b-469b-b68a-f2b5c6091251)

When I try to export it via the Better Export PDF feature, it looks like this:

![Screenshot 2024-03-15 002702](https://github.com/l1xnan/obsidian-better-export-pdf/assets/114219038/b91d8c50-7d59-4398-a0ad-475bf77e9103)

This is the debug-view if it helps:

![Screenshot 2024-03-15 002736](https://github.com/l1xnan/obsidian-better-export-pdf/assets/114219038/45f916f0-fa55-4e70-a032-58b882d7cdc2)

I hope someone can help me.

Thank u :)

**Comment by l1xnan** [comment]
Most of the time the `dataview` renders fine, can you provide a minimal reproduction, I need to debug it to see what the problem is?
